examples/gallery/histograms/scatter_and_histograms.py

```diff
@@ -0,0 +1,64 @@
+"""
+Scatter plot with histograms
+----------------------------
+To create a scatter plot with histograms at the sides of the plot one
+can use :meth:`pygmt.Figure.plot` in combination with
+:meth:`pygmt.Figure.histogram`.
+"""
+
+import numpy as np
+import pygmt
+
+np.random.seed(19680801)
+
+# generate random data
+x = np.random.randn(1000)
+y = np.random.randn(1000)
+
+# get axis limits
+xymax = max(np.max(np.abs(x)), np.max(np.abs(y)))
+
+fig = pygmt.Figure()
+fig.basemap(
+    region=[-xymax - 0.5, xymax + 0.5, -xymax - 0.5, xymax + 0.5],
+    projection="X10c/10c",
+    frame=["WSrt", "a1"],
+)
+
+fillcol = "seagreen"
+
+# plot data points as circles with a diameter of 0.15 centimeters
+fig.plot(x=x, y=y, style="c0.15c", fill=fillcol, transparency=50)
+
+# shift the plot origin and add top margin histogram
+fig.shift_origin(yshift="10.25c")
+
+fig.histogram(
+    projection="X10c/2c",
+    frame=["Wsrt", "xf1", "y+lCounts"],
+    # Give the same value for ymin and ymax to have ymin and ymax
+    # calculated automatically
+    region=[-xymax - 0.5, xymax + 0.5, 0, 0],
+    data=x,
+    fill=fillcol,
+    histtype=0,
+    series=0.1,
+)
```

Comment:
I _personally_ feel adding outlines to the bars makes the histogram look nicer:
![scatter_histo_bars_outline](https://user-images.githubusercontent.com/94163266/224496350-dc21b115-c792-4fa8-abf6-594a80ebb8e1.png)

```suggestion
fig.histogram(
    projection="X10c/2c",
    frame=["Wsrt", "xf1", "y+lCounts"],
    # Give the same value for ymin and ymax to have ymin and ymax
    # calculated automatically
    region=[-xymax - 0.5, xymax + 0.5, 0, 0],
    data=x,
    fill=fillcol,
    pen="0.5p,gray30",
    histtype=0,
    series=0.1,
)
```

Reply:
Then I would prefer white, made a commit on this.

Reply:
I have no too strong preference regarding the outline color. But when looking at the documentation preview, I am a bit concerned that a white outline makes the bars with 1 (or 2) count(s) difficult to see.

Reply:
Since it is only an example to show how to create such a plot I think this is a neglectable detail.

Reply:
OK.